lab2/index.html

```diff
@@ -0,0 +1,123 @@
+						<th style="line-height: 2em;color: black;border: 3px solid #3a4a34; text-align: center;border-radius: 5px;font-weight: bold;" class="TableHeader" onclick="sortTable(2)">Total Price</th>
+						<th style="line-height: 2em;color: black;border: 3px solid #3a4a34; text-align: center;border-radius: 5px;font-weight: bold;" class="TableHeader" onclick="sortTable(3)">Status</th>
+					</tr>
+				</thead>
+				<tbody>
+				</tbody>
+			</table>
+			<div class="actions">
+				<button class="clear_history_button" style="margin-top:10px;">
+						Clear All
+					</button>
+			</div>
+		</section>
+	</main>
+	<footer>
+		<div>
+			<a class="FooterItem" href="menu.html">Term & Conditions</a>
+		</div>
+		<div>
+			<a class="FooterItem" href="menu.html">Privacy Statement</a>
+		</div>
+		<div>
+			<a class="FooterItem" href="menu.html">Cookie Policy</a>
+		</div>
+		<span class="fill"></span>
+		<div>
+			<p class="FooterItemLast">
+				© 2017 Burrito Cafe. All Rights Reserved.
+			</p>
+		</div>
+	</footer>
+	<script>
+		function sortTable(n) {
+			var table, rows, switching, i, x, y, shouldSwitch, dir, switchcount = 0;
+			table = document.getElementById("myTable");
+			switching = true;
+			//Set the sorting direction to ascending:
+			dir = "asc";
+			/*Make a loop that will continue until
+			no switching has been done:*/
+			while (switching) {
+				//start by saying: no switching is done:
+				switching = false;
+				rows = table.getElementsByTagName("TR");
+				/*Loop through all table rows (except the
+				first, which contains table headers):*/
+				for (i = 1; i < (rows.length - 1); i++) {
+					//start by saying there should be no switching:
+					shouldSwitch = false;
+					/*Get the two elements you want to compare,
+					one from current row and one from the next:*/
+					x = rows[i].getElementsByTagName("TD")[n];
+					y = rows[i + 1].getElementsByTagName("TD")[n];
+					/*check if the two rows should switch place,
+					based on the direction, asc or desc:*/
+					if (dir == "asc") {
+						if (x.innerHTML.toLowerCase() > y.innerHTML.toLowerCase()) {
+							//if so, mark as a switch and break the loop:
+							shouldSwitch = true;
+							break;
+						}
+					} else if (dir == "desc") {
+						if (x.innerHTML.toLowerCase() < y.innerHTML.toLowerCase()) {
+							//if so, mark as a switch and break the loop:
+							shouldSwitch = true;
+							break;
+						}
+					}
+				}
+				if (shouldSwitch) {
+					/*If a switch has been marked, make the switch
+					and mark that a switch has been done:*/
+					rows[i].parentNode.insertBefore(rows[i + 1], rows[i]);
+					switching = true;
+					//Each time a switch is done, increase this count by 1:
+					switchcount++;
+				} else {
+					/*If no switching has been done AND the direction is "asc",
+					set the direction to "desc" and run the while loop again.*/
+					if (switchcount == 0 && dir == "asc") {
+						dir = "desc";
+						switching = true;
+					}
+				}
+			}
```

Comment:
Plus, this is just bubble sort. The run time is ... going to be O(n^2).

There are better way to sort array of objects.